.github/workflows/nightly-wheel.yml

```diff
@@ -77,7 +78,7 @@ jobs:
 
       - name: Build the wheel
         run: |
-          MACOSX_DEPLOYMENT_TARGET=10.15 ./csrc/velox/velox/scripts/setup-macos.sh
+          MACOSX_DEPLOYMENT_TARGET=10.15 CPU_TARGET="sse" ./csrc/velox/velox/scripts/setup-macos.sh
```

Comment:
do we also want to set CPU target when build wheel? Also what about Ubuntu wheel?

Reply:
Yes, I am adding cpu target when building wheel. I can make ubuntu wheel also see if required.

Reply:
Fyi ubuntu sse wheel will require creating a new docker image so will create separate pr for that.